src/ducks/balance/components/NoAccount.styl

```diff
@@ -0,0 +1,27 @@
+.NoAccount
+    height calc(50vh - 48px)
```

Comment:
? What is this from ? Is No Account really the half height of the screen ? Shouldn't we have an absolute size ?

Reply:
On the onboarding page, we want the header make 50%, this allow don't have a white space too big after.